deploy/rbac-development.json

```diff
@@ -18,6 +18,7 @@
                 "permissions": [
                     {
                         "actions": [
+                            "Microsoft.Compute/virtualMachines/read",
```

Comment:
we don't have this permission in production, so it shouldn't be added here.

Reply:
If we don't have this, this enrichment will fail either way and is not a valid solution?

Reply:
if it succeeds, it will succeed by virtue of the fact that the RP is an owner on the cluster resource group, not by any subscription permissions it may have.